### ISSUE TITLE
AP is very low during the training

### ISSUE THREAD

**Issue by Qianshaowei** [TWSFar/ResNeSt-mmdetection#1]
Hello, Thank you for sharing this code. When I use this code on mmdetection, I found AP is very low during the training. Do you have this problem？

**Comment by TWSFar** [comment]
> Hello, Thank you for sharing this code. When I use this code on mmdetection, I found AP is very low during the training. Do you have this problem？

No, it's normal

**Comment by Qianshaowei** [comment]
> > Hello, Thank you for sharing this code. When I use this code on mmdetection, I found AP is very low during the training. Do you have this problem？
> 
> No, it's normal

What is the learning rate and weight decay you use? What version of mmdetection are you using? Can you give me the configuration file of the code you ran?I would really appreciate it
